uber/decorators.py

```diff
@@ -315,26 +315,36 @@ def prettify_breadcrumb(str):
 def renderable(func):
     @wraps(func)
     def with_rendering(*args, **kwargs):
-        result = func(*args, **kwargs)
-
         try:
-            result['breadcrumb_page_pretty_'] = prettify_breadcrumb(func.__name__) if func.__name__ != 'index' else 'Home'
-            result['breadcrumb_page_'] = func.__name__ if func.__name__ != 'index' else ''
-        except:
-            pass
+            result = func(*args, **kwargs)
+        except CSRFException as e:
+            message = "Your CSRF token is invalid. Please go back and try again."
+            uber.server.log_exception_with_verbose_context(e)
```

Comment:
minor thing: test this line of code out here, I don't think we need to pass in 'e' here.